idris-settings.el

```diff
@@ -295,8 +295,10 @@ Set to `nil' for no banner."
   "Face for the result of an evaluation in the Idris REPL."
   :group 'idris-repl)
 
-(defcustom idris-repl-history-file "~/.idris/idris-history.eld"
-  "File to save the persistent REPL history to."
+(defcustom idris-repl-history-file nil
+  "File to save the persistent REPL history to.
+
+Defaults to ~/.[:IDRIS_EXECUTABLE_NAME:]/idris-history.eld"
```

Comment:
I'm not sure what is the convention for expressing the `IDRIS_EXECUTABLE_NAME` in the doc so this is just idea.
Happy to change if there is existing convention.

Reply:
erm I do not think there is a specific convention followed. 

 Perhaps this would be better rewritten along the lines as:

> File to save the persistent REPL history to.
>
> By default we assume Idris' default configuration home is $HOME/.idris.
> If you have installed/configured Idris differently, then you may wish to customise this variable.

Note that I have used UNIX conventions for 'home'.

Moreover, I do not think that Idris2 is XDG compliant so have not sought to make mention of that.

Reply:
in fact, I will make the adjustments required.